integ/scripts/bash/stage-deadline.sh

```diff
@@ -8,7 +8,7 @@ set -euo pipefail
 # Check if Deadline assets are staged at configured path
 if [ ! $(ls "$DEADLINE_STAGING_PATH/manifest.json" 2> /dev/null) ]; then
     # Stage Deadline assets
-    npx ../packages/aws-rfdk/bin/stage-deadline --deadlineInstallerURI "s3://thinkbox-installers/Deadline/$DEADLINE_VERSION/Linux/DeadlineClient-$DEADLINE_VERSION-linux-x64-installer.run" --dockerRecipesURI "s3://thinkbox-installers/DeadlineDocker/$DEADLINE_VERSION/DeadlineDocker-$DEADLINE_VERSION.tar.gz" --output "$DEADLINE_STAGING_PATH"
+    npx ../packages/aws-rfdk/bin/stage-deadline --output "$DEADLINE_STAGING_PATH"
```

Comment:
Are we sure we want to always test with the latest version of Deadline? What if we instead make the default behavior to use the latest version of Deadline, but still allow `DEADLINE_VERSION` to be specified? I know there is an existing way to do this with `DEADLINE_STAGING_PATH`, but this would remove that extra manual step.

Regardless of that, there are other references to the `${DEADLINE_VERSION}` variable that would need to be updated/removed.

*   In [`integ/scripts/bash/set-test-variables.sh`](https://github.com/aws/aws-rfdk/blob/58410d21cea1c2ba66a96ba5099ca50ed7da7fcc/integ/scripts/bash/set-test-variables.sh#L30-L33)
*   In `integ/test-config.sh`:
    *   [L10-12](https://github.com/aws/aws-rfdk/blob/0b42aa73418d468ad82b0bf519244589bc835293/integ/test-config.sh#L10-L12)
    *   [L16](https://github.com/aws/aws-rfdk/blob/0b42aa73418d468ad82b0bf519244589bc835293/integ/test-config.sh#L16)
*   In [`integ/scripts/bash/fetch-worker-amis.sh`](https://github.com/aws/aws-rfdk/blob/0b42aa73418d468ad82b0bf519244589bc835293/integ/scripts/bash/fetch-worker-amis.sh#L10)
    *   This one may be harder to remove.
*   In [`integ/README.md`](https://github.com/aws/aws-rfdk/blame/0b42aa73418d468ad82b0bf519244589bc835293/integ/README.md#L13)

Reply:
Looks like you've opted to keep the variable, so I'll resolve this discussion.